README.md

```diff
@@ -81,6 +80,17 @@ All jobs that are using the Known Outgoing IP feature will have one of the follo
 "54.92.235.88"
 ```
 
+Known IPs for the core service (API, etc - to trigger jobs, exchange information on users between CircleCI and Github):
```

Comment:
```suggestion
Known IPs for the core services (used to trigger jobs, exchange information about users between CircleCI and Github etc):
```
~ I was a little confused about what this was trying to say at first. Would this be a bit clearer?